powerfulseal/policy/scenario.py

```diff
@@ -68,6 +69,36 @@ def execute(self):
         self.cleanup()
         return True
 
+    def retry(self, step_action, action_method):
+        ret = False
+        if "retries" in step_action.keys():
+            step_retry = step_action['retries']
+            if "retriesTimeout" in step_retry.keys():
+                final_timeout = step_retry.get('retriesTimeout', {}).get('timeout', 60)
+                retry_sleep = step_retry.get('retriesTimeout', {}).get('sleep', 30)
+                timer = 0
+                while timer < final_timeout:
+                    ret = action_method(schema=step_action)
+                    if ret:
+                        return ret
+                    time.sleep(retry_sleep)
```

Comment:
It would be good to log some messages about retries, etc. For example, like we do [in the http probe](https://github.com/powerfulseal/powerfulseal/blob/a82f1b7a7a5709a787fbb48cb64099e612c666cb/powerfulseal/policy/action_probe_http.py#L116-L134).

Otherwise it will be pretty confusing to see PowerfulSeal just kind of block and not know what's going on. Also applies below.

Reply:
What do you think of the logging statements I added here?

Reply:
I think they'll do just fine!